src/main/java/dev/j3fftw/litexpansion/machine/AdvancedSolarPanel.java

```diff
@@ -100,10 +104,14 @@ private int getGeneratingAmount(@Nonnull Block b, @Nonnull World world) {
         // Note: You need to get the block above for the light check, the block itself is always 0
         if (world.isThundering() || world.hasStorm() || world.getTime() >= 13000
             || b.getLocation().add(0, 1, 0).getBlock().getLightFromSky() != 15
-        )
+        ) {
+            time = "&8Night";
             return this.type.getNightGenerationRate();
-        else
+        }
+        else {
```

Comment:
ew no, don't newline the `else {` add it to the closing bracket

Reply:
ree u and biscuit have different standards

Reply:
I actually follow 1TBS, Cookie is a weirdo